UnityProject/Assets/Scripts/Core/Utils/Utilities.cs

```diff
@@ -7,6 +7,9 @@
 namespace Core.Utils
 {
 
+	public class MindNIPossessingEvent : UnityEvent<Mind, IPlayerPossessable> { }
```

Comment:
this is an odd location for this class. Surely it can go closer to where it is needed, somewhere in the possesion related code.

Reply:
but it's going to be used in many different places in the code, so it makes sense it being generic place?